test/unit/messaging/validate-statement.test.js

```diff
@@ -0,0 +1,48 @@
+const validateStatement = require('../../../app/messaging/validate-statement')
+const mockStatement = require('../../mock-statement-data')
+
+describe('validate statement', () => {
```

Comment:
do you not need to mock `schema` set it to return just `mockStatement` and then have tests around when it throws?